modules/lang-painless/src/main/java/org/elasticsearch/painless/node/SSource.java

```diff
@@ -186,6 +185,20 @@ void analyze(Locals program) {
 
         mainMethod = Locals.newMainMethodScope(scriptInterface, program, reserved.getMaxLoopCounter());
 
+        for (int get = 0; get < scriptInterface.getGetMethods().size();) {
+            String name = scriptInterface.getGetMethods().get(get).getName().substring(3);
+            name = Character.toLowerCase(name.charAt(0)) + name.substring(1);
+            Definition.Type rtn = scriptInterface.getGetReturns().get(get);
+
+            if (reserved.getUsedVariables().contains(name)) {
+                mainMethod.addVariable(new Location("getter [" + name + "]", 0), rtn, name, true);
+                ++get;
+            } else {
+                scriptInterface.getGetMethods().remove(get);
```

Comment:
:) I was a bit lazy here since I knew this code is going to be nuked from orbit at some point.  I will fix it.